bin/strata-cli/src/cmd/change_pwd.rs

```diff
@@ -14,14 +14,13 @@ pub async fn change_pwd(_args: ChangePwdArgs, seed: Seed, persister: impl Encryp
     let term = Term::stdout();
     let mut new_pw = Password::read(true).unwrap();
     let entropy = new_pw.entropy();
-    let _ = term.write_line(format!("Password strength (Overall strength score from 0-4, where anything below 3 is too weak): {}", entropy.score()).as_str());
     if entropy.score() <= Score::Two {
         let _ = term.write_line(
-            entropy
-                .feedback()
-                .expect("No feedback")
-                .to_string()
-                .as_str(),
+            format!(
+                "Password is weak. {}",
+                entropy.feedback().expect("No feedback")
+            )
+            .as_str(),
```

Comment:
Could we consolidate the password checking into a `validate_pass` function that does the error handling and exit when the password is insufficient?